assets/js/modules/pagespeed-insights/components/DashboardPageSpeedWidget.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Dashboard PageSpeed Widget component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { string } from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as MODULES_STORE } from '../../../googlesitekit/modules/datastore/constants';
+import DashboardPageSpeedCTA from './DashboardPageSpeedCTA';
+import DashboardPageSpeed from './DashboardPageSpeed';
+const { useSelect } = Data;
+
+function DashboardPageSpeedWidget( { width, slug } ) {
+	const pagespeedInsightsModule = useSelect( ( select ) => select( MODULES_STORE ).getModule( 'pagespeed-insights' ) );
+	if ( ! pagespeedInsightsModule ) {
+		return null;
+	}
+
+	const { active, setupComplete } = pagespeedInsightsModule;
+	if ( ! active && ! setupComplete ) {
+		return <DashboardPageSpeedCTA />;
+	}
+
+	const { Widget } = global.googlesitekit.widgets.components;
+
+	return (
+		<Widget slug={ slug } width={ width }>
+			<DashboardPageSpeed />
+		</Widget>
+	);
+}
+
+DashboardPageSpeedWidget.propTypes = {
+	width: string,
+	slug: string,
+};
+
+DashboardPageSpeedWidget.defaultProps = {
+	width: '',
+	slug: '',
+};
```

Comment:
```suggestion
```
These don't seem like appropriate defaults. I'd remove them and set the props to be required, as neither should be blank.

Reply:
Updated.